### DISCUSSION TITLE
error:Encountered an error verifying this contract

### DISCUSSION THREAD

**Discussion by akashnikam25** [Cyfrin/foundry-full-course-cu#1654] [Q&A]
I ran the below command 
```
forge script script/DeployFundMe.s.sol:DeployFundMe --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --broadcast --verify --etherscan-api-key $ETHERSCAN_API_KEY -vvvv
```
but I am getting below-error
```
Encountered an error verifying this contract:
Response: `NOTOK`
Details: `Invalid constructor arguments provided. Please verify that they are in ABI-encoded format
```
@Engrpips do you have any idea why this not verified?

**Comment by EngrPips** [comment]
Hello @akashnikam25, The error pointed out  `Invalid constructor arguments`. I would suggest you ensure that the constructor argument is valid. Showing the full output of the Deployment command can also help to give more insight.
